realm-jni/src/io_realm_internal_SharedGroup.cpp

```diff
@@ -159,6 +157,17 @@ JNIEXPORT void JNICALL Java_io_realm_internal_SharedGroup_nativeAdvanceRead
     CATCH_STD()
 }
 
+JNIEXPORT void JNICALL Java_io_realm_internal_SharedGroup_nativeAdvanceReadToVersion
+(JNIEnv *env, jobject, jlong native_ptr, jlong native_replication_ptr, jlong version, jlong index)
```

Comment:
are you referring to `native_replication_ptr`? if yes this need an entire PR to get fix naming in JNI, it's all over the map (check Java_io_realm_internal_SharedGroup_nativeCreate for example)